### PR TITLE
contributor summit 2019 blog post

### PR DESCRIPTION
**Pull request by parispittman**
contributor summit blog post

### CODE REVIEW

**Comment by idealhack** [comment]
https://deploy-preview-13209--kubernetes-io-master-staging.netlify.com/blog/2019/03/14/a-look-back-and-whats-in-store-for-kubernetes-contributor-summits/

images are returning 404s, looks like the paths are case-sensitive

**Comment by idealhack** [comment]
btw we'd better optimize the images a bit -- they are `15 MB` now 😅

otherwise lgtm

File: content/en/blog/_posts/2019-03-14-A-Look-Back-And-Whats-In-Store-For-Kubernetes-Contributor-Summits.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+---
+title: A Look Back and What's in Store for Kubernetes Contributor Summits
+date: 2019-03-14
+---
+
+**Authors:**
+Paris Pittman (Google), Jonas Rosland (VMWare)
```
Comment (jonasrosland) [inline content/en/blog/_posts/2019-03-14-A-Look-Back-And-Whats-In-Store-For-Kubernetes-Contributor-Summits.md]: It’s VMware, not VMWare :)

**Pull request merged**